editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -59,8 +59,8 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(475) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(485)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThan(620) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
this is sub-ideal!

Reply:
I know.. I'm still trying to run tests locally after seeing all kind of globalThis errors and my direnv had to be reinstalled

Reply:
fixed